libpod/runtime_img.go

```diff
@@ -47,6 +48,8 @@ var (
 	// DirTransport is the transport for pushing and pulling
 	// images to and from a directory
 	DirTransport = "dir"
+	// TransportNames are the supported transports in string form
+	TransportNames = [...]string{DefaultRegistry, DockerArchive, OCIArchive, "ostree:", "dir:"}
```

Comment:
Should we get this from containers/image?

Reply:
I didnt see it defined there like this.

Reply:
It would be good if we could grab it from there (if not now, then at some point in the future) - otherwise might be hard to keep them in sync

Reply:
Look for Transports.


	"github.com/containers/image/transports/alltransports"
transports.ListTransports()

Reply:
done